control/item_manager.py

```diff
@@ -75,41 +75,46 @@ def create_loot_box(self, guild_id: int) -> LootBox:
         ]
 
         weights = [self.get_item(guild_id, x).cost for x in item_pool]
-        chance_for_item = self.settings_manager.get_setting(
-            guild_id,
-            SettingsManager.BEANS_SUBSETTINGS_KEY,
-            SettingsManager.BEANS_LOOTBOX_RARE_CHANCE_KEY,
-        )
+
+        # Spawn Chances
         mimic_chance = 0.1
-        chance_for_bonus_beans = 0.2
+        chance_for_item = 0.2
+        medium_chest_chance = 0.2
+        large_chest_chance = 0.01
         random_item = None
 
-        min_beans = self.settings_manager.get_setting(
-            guild_id,
-            SettingsManager.BEANS_SUBSETTINGS_KEY,
-            SettingsManager.BEANS_LOOTBOX_MIN_BEANS_KEY,
-        )
-        max_beans = self.settings_manager.get_setting(
-            guild_id,
-            SettingsManager.BEANS_SUBSETTINGS_KEY,
-            SettingsManager.BEANS_LOOTBOX_MAX_BEANS_KEY,
-        )
-
-        beans = random.randint(min_beans, max_beans)
+        # Chest Ranges
+        small_min_beans = 40
+        small_max_beans = 80
+        medium_min_beans = 200
+        medium_max_beans = 300
+        large_min_beans = 700
+        large_max_beans = 900
+        small_beans_reward = random.randint(small_min_beans, small_max_beans)
```

Comment:
you dont need to pre roll all 3 outcomes, its better to roll only the one you need once it gets chosen.